styles/admin.css

```
@@ -0,0 +1,516 @@
+  }
+
+  .admin-search_input {
+    width: 100%;
+    border-width: 0;
+    box-shadow: none;
+    background-color: transparent;
+    outline: none;
+
+    &:focus {
+      outline: none;
+      ring: 0;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)